### ISSUE TITLE
Error building the documentation

### ISSUE THREAD

**Issue by joequant** [odoo/documentation-user#189]
Getting the following error building the documentaiton

```
reading sources... [  4%] accounting/bank/setup/create_bank_account
Exception in thread fields_get fetcher thread 0/4:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 754, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/joe/rpmbuild/BUILD/odoo-10.0-20161011/documentation-user-af4a3a1f6a83b0e051f3fc34a8547e046a2766f3/_extensions/demo_link.py", line 170, in _fetch_fields
    view_id = next((id_ for type, id_ in action['views'] if type == task.view), False)
TypeError: list indices must be integers, not str

reading sources... [  4%] accounting/bank/setup/foreign_currency
reading sources... [  4%] accounting/bank/setup/manage_cash_register

```

**Comment by afwanwh** [comment]
I have same issue here

**Comment by tivisse** [comment]
Hello @joequant, @afwanwh  it should have been fixed in 10.0 by this commit https://github.com/odoo/documentation-user/commit/9bfc3e7eb6941273cb0e6e8cbff88c83c3f855c0

I just backported the fix in 9.0 at https://github.com/odoo/documentation-user/commit/e2a425c7c3ae62d1d841ca6eec626107bcfb15fe

Have a nice day.
